soroban-sdk/src/bigint.rs

```diff
@@ -1094,15 +1094,13 @@ impl BigInt {
     /// Returns true if the [BigInt] is zero.
     pub fn is_zero(&self) -> bool {
         let env = self.env();
-        let is_zero = env.bigint_is_zero(self.0.to_object());
-        bool::try_from(is_zero).unwrap()
+        env.bigint_is_zero(self.0.to_object()).is_true()
     }
 
     /// Returns the minimum number of bits required to store the [BigInt].
-    pub fn bits(&self) -> u32 {
+    pub fn bits(&self) -> u64 {
         let env = self.env();
-        let bits = env.bigint_bits(self.0.to_object());
-        u32::try_from(bits).unwrap()
+        env.bigint_bits(self.0.to_object())
```

Comment:
@jayz22 @graydon Why does the `bigint_bits` function return `u64`? In Rust built-in primitive types like u32, u64, all define their number of bits in `u32`. I find it highly unlikely our BigInt type would realistically be able to hold more than u32::MAX bits either.

Reply:
Reposted this question at https://github.com/stellar/rs-soroban-env/issues/423. Let's discuss there.